config/routes.yml

```diff
@@ -0,0 +1,7 @@
+ps_democqrshooksusage_toggle_is_allowed_for_review:
```

Comment:
We could add in comment a link to the doc that explains how to add routes to modern modules